bin/pwa.dart

```diff
@@ -103,24 +103,24 @@ Future _generateManifestJson(String webDir, Map pubspec) async {
   await _updateIfNeeded(indexHtmlFile.path, indexHtmlContent);
 }
 
-/// If build/web is empty, run `pub build`.
+/// If build is empty, run `pub build`.
 Future _buildProjectIfEmptyOrOld(List<String> sources, List<String> excludes,
     List<String> offlineDirs) async {
   // This works only with the default value.
-  if (offlineDirs.length == 1 && offlineDirs.first == 'build/web') {
-    Directory dir = new Directory('build/web');
+  if (offlineDirs.length == 1 && offlineDirs.first == 'build') {
+    Directory dir = new Directory('build');
     if (dir.existsSync() && dir.listSync().isNotEmpty) {
       int lastSourceTimestamp = _getLastTimestamp(sources, excludes);
       int lastOfflineTimestamp = _getLastTimestamp(offlineDirs, null);
       if (lastSourceTimestamp < lastOfflineTimestamp) return;
     }
-    print('Running pub build:');
-    String executable = 'pub';
+    print('Running webdev build:');
+    String executable = 'webdev';
```

Comment:
Does this assume `pub global activate webdev` or something else? If it does, please document it in the readme.

Also add a TODO to make it work with non-global webdev (where people add it to the `dev_dependencies:`.

Reply:
As stated in the webdev repository [README](https://github.com/dart-lang/webdev#installation):
> webdev is not meant to be used as a dependency. Instead, it should be "activated".
So I think it won't be necessary to add the steps to use it as a dependency. For the other case, I will add a warning in the README, to activate webdev previously.

Reply:
Added an advice in README to get webdev installed previously.